core/foundation/inc/ROOT/RWrap_libcpp_string_view.h

```diff
@@ -1,591 +0,0 @@
-// -*- C++ -*-
```

Comment:
GitHub tells me that this file got emptied but isn't deleted. Is that intentional? Or a bug in GitHub's interface?

Reply:
![gh_iface_bug](https://github.com/root-project/root/assets/6578603/8ed2af11-0e4e-4a86-b09e-4e868ce158fb)

For me it shows it like this, so correctly deleted. Maybe a bug in the comparison interface